doc/gui/examples/charts/histogram-nbins.py

```diff
@@ -18,7 +18,7 @@
 from taipy.gui import Gui
 
 # Random set of 100 samples
-samples = {"x": [random.gauss() for i in range(100)]}
+samples = {"x": [random.gauss(mu=0.0, sigma=1.0) for _ in range(100)]}
```

Comment:
IS it a new rule to make the default parameter values explicit?

Reply:
This is a really weird function. So the `random.gauss()` required the `mu` and `sigma`. However, if None, the value will be `0.0` and `1.0`.

But since they are required, mypy will complain.

Reply:
OK thx.